src/connect-store-node.md

```diff
@@ -0,0 +1,52 @@
+# Connect external store node
```

Comment:
Does this render the book correctly? Since its a new page not sure if you need to add it to https://github.com/waku-org/waku-simulator/blob/master/src/SUMMARY.md

Reply:
I think it renders correctly. Summary updated in https://github.com/waku-org/waku-simulator/pull/87/commits/c997a8c18a8c6d00c66e75f2f72f25e943680855